rllib/models/torch/torch_action_dist.py

```diff
@@ -305,3 +309,95 @@ def sample(self):
     @override(ActionDistribution)
     def required_model_output_shape(action_space, model_config):
         return np.prod(action_space.shape)
+
+
+class TorchMultiActionDistribution(TorchDistributionWrapper):
+    """Action distribution that operates for list of (flattened) actions.
```

Comment:
This doesn't need to be flat any more right? So one of the child distributions could be in turn a MultiActionDist.

Reply:
Correct. I'll fix the comment.

Reply:
Done.